rest/route/task.go

```diff
@@ -94,12 +94,17 @@ func (tgh *taskGetHandler) Run(ctx context.Context) gimlet.Responder {
 	if err != nil {
 		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "API model error"))
 	}
-
 	err = taskModel.BuildFromService(tgh.url)
 	if err != nil {
 		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "API model error"))
 	}
 
+	err = taskModel.GetAMI()
+	if err != nil {
+		return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "API model error"))
+	}
+	taskModel.GetProjectIdentifier()
```

Comment:
I think this is a bigger thing to remove actually 😓  we may need to add this to all the task routes.

I wonder if a prettier way to do this would be to have a BuildWithArgs() function with a variable "includeNonTaskFields", that calls both BuildFromService nad GetProjectIdentifier/GetAMI, and everywhere BuildFromService is used in the rest route we actually call BuildWithArgs (so that GetAMI and GetProjectIdentifier is also included).  That name isn't ideal but idk.

Alternatively you can just look at everywhere BuildFromService is used in rest/route for APITask and add GetProjectIdentifier.

Reply:
Done! I did the alternate way :)